vembrane/__init__.py

```diff
@@ -130,6 +139,13 @@ def main():
         help="Keep all annotations of a variant if at least one of them passes "
         "the expression.",
     )
+
+    parser.add_argument(
+        "--rename",
+        default="=",
+        help="Replace ANN field names with desired variables. "
+        "Multiple replacements need to be comma-separated.",
+    )
```

Comment:
1. I'd personally prefer to use the `type` argument of argparse, e.g.:
```python
    def mapping(value: str) -> Dict[str, str]:
        return dict(map(str.strip, v.strip().split("=")) for v in value.split(',') if v)

    parser.add_argument(
        "--rename",
        default="=",
        type=mapping,
        help="Replace ANN field names with desired variables. "
        "Multiple replacements need to be comma-separated.",
    )
```
so that parsing is already done when you access `args.rename`

2. the default being "=" looks weird :D

Reply:
oh nice, thats really cool